Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+JavaBindingsPrinting.swift

```diff
@@ -72,6 +74,11 @@ extension JNISwift2JavaGenerator {
     printImports(&printer)
 
     printModuleClass(&printer) { printer in
+      if config.effectiveMemoryManagementMode.requiresGlobalArena {
+        printer.print("static final SwiftArena \(globalArenaName) = SwiftArena.ofAuto();")
```

Comment:
If we're doing that it should be living in SwiftKitCore and not in the generated sources as this way you'd have one per module, but we'd want one global one I think